libs/financial-aid/shared/src/data/serviceCenters.ts

```diff
@@ -0,0 +1,644 @@
+export interface ServiceCenter {
+  name: string
+  number: number
+  phone: string
+  address: string
+  addressPostalCode: string
+  postalCodes: number[]
+  active?: boolean
+  link?: string
+}
+
+// Compiled from https://www.samband.is/sveitarfelogin/ list and
+// https://is.wikipedia.org/wiki/Íslensk_sveitarfélög_eftir_sveitarfélagsnúmerum
+
+export const serviceCenters: ServiceCenter[] = [
+  {
```

Comment:
Is this as well on the roadmap to get dynamically from some source registry?

Reply:
Yes as well, waiting for Þjóðskrá 🙃